x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorService.java

```diff
@@ -44,10 +48,27 @@ public class SlmHealthIndicatorService implements HealthIndicatorService {
         null
     );
 
+    public static final String ACTION_CHECK_RECENTLY_FAILED_SNAPSHOTS_HELP_URL = null;
```

Comment:
I think we can already provide an alias that points to the generic `fix common cluster issues` url ( https://www.elastic.co/guide/en/elasticsearch/reference/current/fix-common-cluster-issues.html#fix-common-cluster-issues )
The diagnosis URL will soon be non-nullable